### ISSUE TITLE
Code of conduct File 

### ISSUE THREAD

**Issue by akrish4** [Akashsindhu/AIes_object_detection#6]
Hi @Akashsindhu @tommychavez 
I would like to contribute by working on a code of conduct file
kindly assign me 

Thanks

**Comment by akashsindhu96** [comment]
Closing this issue. You can re-open.

**Comment by akrish4** [comment]
> Closing this issue. You can re-open.

why have u closed it ? I can't reopen it since it is closed by u, pls reopen and close once the pr is merged @Akashsindhu

**Comment by akrish4** [comment]
Kindly add the labels `DWOC` and  `Level-3` or `Level-2` or `Level-1` to issue  @Akashsindhu 
as this project is registered under dwoc

**Comment by akrish4** [comment]
Hi @Akashsindhu 
I have worked on it , kindly do merge it 
Kindly do add the same labels 
#8

**Comment by akrish4** [comment]
@Akashsindhu  Kindly do check and merge it as soon as possible so that i can work on the next issue and open another pr for it

**Comment by akashsindhu96** [comment]
@akrish4 PR is merged. Closing this issue, reopen if you have any update.

**Comment by akrish4** [comment]
Hi @Akashsindhu 
Kindly do add the same labels to the PR aswell
